src/package.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "example-app",
+    "private": true,
+    "type": "module",
+    "scripts": {
+        "dev": "vite",
+        "build": "vite build",
+        "lint": "eslint .",
+        "lintf": "eslint . --fix"
+    },
+    "dependencies": {
+        "@inertiajs/vue3": "^1.0.7",
+        "@tailwindcss/forms": "^0.5.3",
+        "@tailwindcss/typography": "^0.5.9",
+        "laravel-vite-plugin": "^0.7.8",
+        "lodash": "^4.17.21",
+        "tailwindcss": "^3.3.2",
+        "vue": "^3.3.4"
+    },
+    "devDependencies": {
+        "@blumilksoftware/eslint-config": "^2.0.0",
+        "@typescript-eslint/eslint-plugin": "^5.59.8",
+        "@vitejs/plugin-vue": "^4.2.3",
+        "autoprefixer": "^10.4.14",
+        "eslint": "^8.41.0",
+        "eslint-config-prettier": "^8.8.0",
+        "eslint-plugin-tailwindcss": "^3.12.1",
+        "eslint-plugin-vue": "^9.14.1",
+        "postcss": "^8.4.24",
+        "vite": "^4.3.9"
+    }
```

Comment:
Did you run
`npm outdated`
and
`npm update --save` ?

Reply:
updated